app/grandchallenge/forums/templates/forum/forum_detail.html

```diff
@@ -0,0 +1,98 @@
+{% extends 'board_base.html' %}
+{% load i18n %}
+{% load mptt_tags %}
+{% load forum_tags %}
+{% load forum_conversation_tags %}
+{% load forum_permission_tags %}
+{% load forum_tracking_tags %}
+{% load activity_tags %}
+{% load forum_extras %}
+
+
+{% block sub_title %}{{ forum.name }}{% endblock sub_title %}
+
+{% block content %}
+<div class="row">
+  <div class="col-12">
+    <h1>{{ forum.name }}</h1>
+  </div>
+</div>
+{% if sub_forums %}
+<div class="row">
+  <div class="forumlist-top-controls col-12">
+    {% if request.user.is_authenticated %}
+    <div class="float-right">
+      <a href="{% url 'forum_tracking:mark_subforums_read' forum.pk %}" class="d-inline-block"><i class="fa fa-check-circle-o">&nbsp;</i>{% trans "Mark subforums read" %}</a>
+    </div>
+    {% endif %}
+  </div>
+</div>
+{% forum_list sub_forums %}
+{% endif %}
+{% if forum.is_forum %}
+<div class="mt-4 mb-3 row">
+  <div class="col-6 col-md-4 forum-actions-block">
+    {% get_permission 'can_add_topic' forum request.user as user_can_add_topic %}
+    {% get_permission 'can_read_forum' forum request.user as user_can_read_forum %}
+    {% if user_can_add_topic %}
+    <a href="{% url 'forum_conversation:topic_create' forum.slug forum.pk %}" class="btn btn-primary btn-sm"><i class="fa fa-comments fa-lg"></i>&nbsp;{% trans "New topic" %}</a>
+    {% endif %}
+    {% if user_can_read_forum and request.user|is_following:forum %}
+        {% get_follow_object_pk request.user forum as follow_object_pk %}
+        <form style="display: inline;" method="post" action="{% url 'notifications:follow-delete' pk=follow_object_pk %}">
+            {% csrf_token %}
+            <button type="submit" class="btn btn-info btn-sm btn-subscription"><i class="fas fa-check"></i>
+                {% trans "Unsubscribe" %}
+            </button>
+        </form>
+    {% elif user_can_read_forum and not request.user|is_following:forum %}
+        <form style="display: inline;" method="post" action="{% url 'notifications:follow-create' %}">
+            {% csrf_token %}
+            <input name="content_type" type="hidden" value="{{ forum|get_content_type }}">
+            <input name="actor_only" type="hidden" value="False">
+            <input name="object_id" type="hidden" value="{{ forum.id }}">
+            <input name="user" type="hidden" value="{{ request.user.id }}">
+            <button type="submit" class="btn btn-info btn-sm btn-subscription"><i class="fas fa-check"></i>
+                {% trans "Subscribe" %}
+            </button>
+        </form>
+    {% endif %}

```

Comment:
General comment - do not construct forms yourself in templates, Django Forms are designed to do this and are much easier to change, re-use and test.